Sources/App/Extensions/Application+Static.swift

```diff
@@ -9,12 +9,24 @@ import Vapor
 
 extension Application {
 
-    static let baseUrl = Environment.get("BASE_URL")!
-    static let databaseUrl = URL(string: Environment.get("DB_URL")!)!
+    static private func fetchEnv(key: String) -> String {
```

Comment:
Can we turn this an extension on the Environment object?
IMHO: it would make sense to use: Environment.fetch(key: "key") instead of a fetchEnv function.

Reply:
@tib Updated. I squashed my commits. So, you’ll see it in the chaged files.